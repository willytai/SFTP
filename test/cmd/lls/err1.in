lls -l dirforcheck/fuck
lls -l dirforcheck/shit
lls -l nosuchfile
lls -l ../thisisbullshit
lls -l ../thisischianger
