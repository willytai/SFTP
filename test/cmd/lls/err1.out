lls: dirforcheck/fuck: No such file or directory
lls: dirforcheck/shit: No such file or directory
lls: nosuchfile: No such file or directory
lls: ../thisisbullshit: No such file or directory
lls: ../thisischianger: No such file or directory
